src/github/handlers/index.ts

```diff
@@ -19,6 +20,7 @@ function tryCatchWrapper(fn: (event: EmitterWebhookEvent) => unknown) {
 
 export function bindHandlers(eventHandler: GitHubEventHandler) {
   eventHandler.on("repository_dispatch", repositoryDispatch);
+  eventHandler.on("issue_comment.created", issueCommentCreated);
   eventHandler.onAny(tryCatchWrapper((event) => handleEvent(event, eventHandler))); // onAny should also receive GithubContext but the types in octokit/webhooks are weird
 }
```

Comment:
The kernel should process all incoming events dynamically? 

For example, Telegram and Google Drive webhooks as well. I don't think its a good approach to list every event.

Reply:
From what I've seen these events are deduced from Octokit events. Eventually we could extend them to other web-hooks I suppose?
But this is just because we need to do something specific on that event within the kernel itself (this one is used for showing the `/help` command).